pkg/ingester/ingester_test.go

```diff
@@ -5159,7 +5159,7 @@ func TestIngester_QueryStream(t *testing.T) {
 
 	// Query back the series using GRPC streaming.
 	inst := ring.InstanceDesc{Id: "test", Addr: listener.Addr().String()}
-	c, err := client.MakeIngesterClient(inst, defaultClientTestConfig(), client.NewMetrics(nil))
+	c, err := client.MakeIngesterClient(inst, defaultClientTestConfig(), client.NewMetrics(nil), nil)
```

Comment:
I would update the tests adding a `log.NewNopLogger()` instead of a nil, that's more correct and still easy enough.